ParisBNF/abb/BNFabb57.xml

```diff
@@ -189,124 +190,136 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <additions>
               <list>
                 <item xml:id="a1">
-                  <locus target="#1r"/>
-                  <desc type="OwnershipNote" xml:lang="gez"/>
+                  <locus target="#31rb"/>
+                  <desc type="OwnershipNote" xml:lang="gez">An extended ownership note, mentioning the
```

Comment:
xml:lang="gez" here can be deleted (your text is in English, and the reference to the language of the note in q is given already below)
Also below

Reply:
ok